### PR TITLE
[WEB-7423] Trigger new SendGrid emails for Studio "One month free" and "$1 for your first month" offers

### PR DESCRIPTION
**Pull request by ricjohn-acosta**

### CODE REVIEW

**Review by luke-wyeth — APPROVED** [review]
LGTM. great work

File: tests/MailerTest.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function testDefaultEmailConfiguration()
 
         # Check number of email templates
         # Change this number when a new template is added
-        $this->assertEquals(62, count($emailOptions));
+        $this->assertEquals(72, count($emailOptions));
```
Comment (ben-farrelly) [inline tests/MailerTest.php:29]: Why has this jumped up 10 with the addition of two emails?

Comment (ricjohn-acosta) [reply tests/MailerTest.php:29]: I think it hasn't been updated in a while - I counted the emails in the config and it's 72

Comment (ben-farrelly) [reply tests/MailerTest.php:29]: Ahh cool cool. Yeah looks like it hasn't been updated for a wee while.

**Review by ben-farrelly — APPROVED** [review]
Nice one @ricjohn-acosta - let me know if you need a hand with rolling out a new version of this package

**Pull request merged**